n3fit/src/n3fit/layers/msr_normalization.py

```diff
@@ -93,6 +114,14 @@ def call(self, pdf_integrated, photon_integral):
             ]
         if self._vsr_enabled:
             numerators += [self.vsr_factors]
+        if self._csr_enabled:
+            for i in range(0, len(CSR_COMPONENTS)):
+                numerators += [
+                    op.batchit(4.0 - 1./3. * y[IDX['v15']], batch_dimension=0)
+                ]
+            numerators += [self.vsr_factors]
```

Comment:
I'm confused about this part.
Why would this be different from simply doing:

```python
 numerators = len(CSR_COMPONENTS)*[op.batchit(4.0 - 1./3. * y[IDX['v15']], batch_dimension=0)]
```

Reply:
yes, you re absolutely right